### PR TITLE
#2382 Optimizations for "How to split into chunks flexible content fields"

### PR DESCRIPTION
**Pull request by gchtr**
**Ticket**: #2382

I added some optimizations for #2368 

- Moved new content from section about Reapeater Fields to Flexible Content Field section.
- Used `include()` function instead of `include` statement, as this is recommended by Twig, see https://twig.symfony.com/doc/3.x/tags/include.html.
- Added some syntax optimizations.

### CODE REVIEW

**Comment by 19h47** [comment]
Didn't know about the `include` function from Twig 3 either, thank you! 😮

**Comment by gchtr** [comment]
> Didn't know about the `include` function from Twig 3 either, thank you! 😮

It’s also available in Twig 2: https://twig.symfony.com/doc/2.x/functions/include.html.

File: docs/v2/integrations/advanced-custom-fields.md [hunk 1]
```diff
@@ -232,7 +222,25 @@ Similar to nested repeaters, you should only call the `meta` method once when yo
 {% endfor %}
 ```
 
-* * *
+### Split Flexible Content Fields into includes / chunks
+
+We can break Flexible Content Fields into small blocks with include files utilizing the `acf_fc_layout` value. This way you have more flexibility and reusability for included sections. For instance, this is a great way to build landing pages that re-use the same blocks in different configurations.
+
+Assuming your block Twig files are inside a **blocks** folder:
```
Comment (jarednova) [inline docs/v2/integrations/advanced-custom-fields.md:229]:
```suggestion
Assuming your block Twig files are inside a **my_blocks** folder within your other Twig file directory (ex: `my_theme/views/my_blocks`).
```

File: docs/v2/integrations/advanced-custom-fields.md [hunk 2]
```diff
@@ -232,7 +222,25 @@ Similar to nested repeaters, you should only call the `meta` method once when yo
...
+
+```twig
+{% for block in post.meta( 'blocks' ) %}
+    {{ include("blocks/#{ block.acf_fc_layout|sanitize ) }.twig", {
```
Comment (jarednova) [inline docs/v2/integrations/advanced-custom-fields.md:233]:
```suggestion
    {{ include("my_blocks/#{ block.acf_fc_layout|sanitize ) }.twig", {
```

File: docs/v2/integrations/advanced-custom-fields.md [hunk 3]
```diff
@@ -232,7 +222,25 @@ Similar to nested repeaters, you should only call the `meta` method once when yo
...
+        block: block
+    }) }}
+{% endfor %}
+```
+
+The [sanitize](https://timber.github.io/docs/v2/guides/filters/#sanitize) filter will slugify the block name. Consider this example for a Flexible Content Field named `credit` containing a `text` field, inside the **blocks/photo.twig** file:
```
Comment (jarednova) [inline docs/v2/integrations/advanced-custom-fields.md:239]:
```suggestion
The [sanitize](https://timber.github.io/docs/v2/guides/filters/#sanitize) filter will slugify the block name. Consider this example for a Flexible Content Field named `credit` containing a `text` field, inside the **my_blocks/photo.twig** file:
```

**Review by jarednova — CHANGES_REQUESTED** [review]
Looks good! Just one suggestion in regards to use of names. If I were building a theme, I'd probably keep everything named the same ("blocks") — but I think we should make super-clear to readers that there's no "magic" in naming things the same and they can use whatever names they need/want.

**Comment by 19h47** [comment]
Totally agreed. No "where do you put this folder" question

**Comment by gchtr** [comment]
@jarednova I agree, we can make this clearer, but I’m always a little hesitant about using "my" in examples. I wouldn’t use `my_blocks` as a name for anything that I code myself. I’m sure it adds to the notion of "you can actually choose a name for this", but I think it’s not good practice for actually naming things. I also see that lots of developers copy-paste code from the documentation as it is. So I’d rather use real world code examples than examples where you’d have to rename stuff before using it.

But, maybe I’m all wrong 😊. Maybe if I’ve yet missed that using "my" is some best practice that’s established in the developer community.

> but I think we should make super-clear to readers that there's no "magic" in naming things the same and they can use whatever names they need/want.

Wouldn’t it be clear enough if we mention that you can name the folder as you want? Maybe like this:

`Assume your block Twig files are inside a folder within your other Twig template directory. For example, if you used a folder named **blocks**, then that would be the **wp-content/themes/example-theme/views/blocks** folder.`

**Comment by jarednova** [comment]
@gchtr I definitely take `my_` as a "you can actually choose a name for this, but I think it’s not good practice for actually naming things." Though, agreed on the copy-and-pasted tendencies out there. As long as the "magic" problem is solved for, your solution of `Assume your block Twig files are inside a folder within your other Twig template directory. For example, if you used a folder named **blocks**, then that would be the **wp-content/themes/example-theme/views/blocks** folder` is totally 👍  w me!

**Comment by gchtr** [comment]
@jarednova Cool, so I guess we’re on the same page here. I’ve updated the documentation once again a little in [`24c790a` (#2388)](https://github.com/timber/timber/pull/2388/commits/24c790ac33038add222bd381dde71eaaebfd4fe9). I hope it’s clearer now, even without using "my".

**Review by jarednova — APPROVED** [review]
LGTM!

**Pull request merged**